### PR TITLE
Mention source of `elements` database in docstring

### PR DESCRIPTION
**Pull request by ericpre**
As mentioned in https://github.com/hyperspy/exspy/discussions/85#discussioncomment-10678716.

### Progress of the PR
- [x] Mention source of `elements` database in docstring,
- [x] docstring updated (if appropriate),
- [x] update user guide (if appropriate),
- [n/a] added tests,
- [x] add a changelog entry in the `upcoming_changes` folder (see [`upcoming_changes/README.rst`](https://github.com/hyperspy/exspy/blob/main/upcoming_changes/README.rst)),
- [x] Check formatting of the changelog entry (and eventual user guide changes) in the `docs/readthedocs.org:exspy` build of this PR (link in github checks)
- [x] ready for review.

### CODE REVIEW

File: doc/reference/material.rst [hunk 1]
```diff
@@ -32,32 +34,50 @@ Database of element properties. The following properties are included:
     └── Physical_properties
         └── density_gcm3
 
-Examples:
+The X-ray lines energies are taken from Chantler et al. [1]_.
 
-.. code::
+The line weight, more precisely the approximate line weight for K,L M
+shells are taken from Electron Probe Quantitation (EPQ) library [2]_.
```
Comment (jlaehne) [inline doc/reference/material.rst:40]:
```suggestion
The line weight, more precisely the approximate line weight for K, L, M
shells are taken from Electron Probe Quantification (EPQ) library [2]_.
```

File: exspy/_misc/elements.py [hunk 2]
```diff
@@ -4996,6 +4996,14 @@
     └── Physical_properties
         └── density_gcm3
 
+The X-ray lines energies are taken from Chantler et al. [1]_.
+
+The line weight, more precisely the approximate line weight for K,L M
+shells are taken from Electron Probe Quantitation (EPQ) library [2]_.
```
Comment (jlaehne) [inline exspy/_misc/elements.py:5002]:
```suggestion
The line weight, more precisely the approximate line weight for K, L, M
shells are taken from Electron Probe Quantification (EPQ) library [2]_.
```

**Comment by jlaehne** [comment]
Chantler reference is already mentioned in other docstrings, but way of mentioning is quite inconsistent.

**Comment by ericpre** [comment]
> Chantler reference is already mentioned in other docstrings, but way of mentioning is quite inconsistent.

@jlaehne, is it better now with the references?

File: exspy/_misc/elements.py [hunk 3]
```diff
@@ -4996,6 +4996,14 @@
     └── Physical_properties
         └── density_gcm3
 
+The X-ray lines energies are taken from Chantler et al. [1]_.
+
+The line weight, more precisely the approximate line weight for K, L, M
+shells are taken from Electron Probe Quantification (EPQ) library [2]_.
```
Comment (jlaehne) [inline exspy/_misc/elements.py:5002]:
```suggestion
shells are taken from the Electron Probe Quantification (EPQ) library [2]_.
```

File: doc/reference/material.rst [hunk 4]
```diff
@@ -32,32 +34,50 @@ Database of element properties. The following properties are included:
     └── Physical_properties
         └── density_gcm3
 
-Examples:
+The X-ray lines energies are taken from Chantler et al. [1]_.
 
-.. code::
+The line weight, more precisely the approximate line weight for K, L, M
+shells are taken from Electron Probe Quantification (EPQ) library [2]_.
```
Comment (jlaehne) [inline doc/reference/material.rst:40]:
```suggestion
shells are taken from the Electron Probe Quantification (EPQ) library [2]_.
```

File: doc/reference/material.rst [hunk 5]
```diff
@@ -32,32 +34,50 @@ Database of element properties. The following properties are included:
...
+
+The field ``threshold`` and ``edge`` are taken from Gatan EELS atlas
```
Comment (jlaehne) [inline doc/reference/material.rst:42]:
```suggestion
The field ``threshold`` and ``edge`` are taken from the Gatan EELS atlas
```

File: exspy/_misc/elements.py [hunk 6]
```diff
@@ -4996,6 +4996,14 @@
...
+
+The field ``threshold`` and ``edge`` are taken from Gatan EELS atlas
```
Comment (jlaehne) [inline exspy/_misc/elements.py:5004]:
```suggestion
The field ``threshold`` and ``edge`` are taken from the Gatan EELS atlas
```

**Comment by jlaehne** [comment]
> > Chantler reference is already mentioned in other docstrings, but way of mentioning is quite inconsistent.
> 
> @jlaehne, is it better now with the references?

Maybe align also the citation in the Bibliography of the User Guide (use DOI, it currently uses the subtitle and not the title of the page).

**Pull request merged**